backend/src/API/teamEventsAPI.ts

```diff
@@ -75,3 +76,25 @@ export const getAllTeamEventsForMember = async (
   email: string,
   isPending: boolean
 ): Promise<TeamEventInfo[]> => TeamEventsDao.getTeamEventsForMember(email, isPending);
+
+export const updateTeamEventAttendance = async (
+  teamEventAttendance: TeamEventAttendance,
+  user: IdolMember
+): Promise<TeamEventAttendance> => {
+  if (!PermissionsManager.canEditTeamEvent(user)) {
```

Comment:
Can you await this, store the result in a variable, then check the value of what the promise resolves to? Right now you are checking the truthy/falsy value of the promise, not the value itself.

Reply:
We messed this up above in the past as well. It would be awesome if you could fix it in this file too, even for methods that are not part of your change.

Reply:
That's not in the scope of the PR.

Reply:
ahh makes sense, maybe the rest in another PR?